invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -26,7 +26,16 @@ class AffiliationsSearchOptions(SearchOptions):
     """Search options."""
 
     suggest_parser_cls = SuggestQueryParser.factory(
-        fields=["name^100", "acronym^50", "title.*^20", "id^20", "aliases^20"],
+        fields=[
+            "name.suggest^100",
+            "name^80",
```

Comment:
Does it make sense to have both the `name.suggest` and `name` here?

Same applies for `acronym`